gfauto/whitelist.dic

```diff
@@ -172,3 +172,6 @@ vec2
 vec4
 framebuffer
 gfautotests
+prefix0
```

Comment:
One exception to what I said: please prefer splitting words wherever possible (even when ending with a number). I think this is a Python convention, or maybe just something I adopted. So please change to prefix_0, prefix_1, and remove these lines from the whitelist.